### DISCUSSION TITLE
how allow cors * in node?

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [subquery/subql#1299] [Q&A]
when I start node, what parameter or env var to pass into subql node to make it accept requests from any origin domain?

need to access subql running in codespace

**Comment by stwiname** [comment]
Hi @dzmitry-lahoda, cors is only enabled for the query service. The node should not be made public so making requests shouldn't require cors

**Reply by dzmitry-lahoda** [reply]
ok, thanks. will fix.

**Reply by tolgaisik** [reply]
Excuse me. I do not understand the explanation. Why cant i send gql request to localhosted docker image ? Getting a cors error. Thank you.

**Comment by dzmitry-lahoda** [comment]
hm, so it is not cors here https://github.com/subquery/subql/blob/main/packages/query/src/yargs.ts

**Comment by dzmitry-lahoda** [comment]
https://github.com/subquery/subql/blob/6c3c928c0844b96e207a2afb78840eafbe1df199/packages/query/src/graphql/graphql.module.ts#L138 oh it may be default

**Comment by tolgaisik** [comment]
i get a cors error for localhost graphql request.
